docs/interpreter/jdbc.md

```diff
@@ -507,8 +507,9 @@ Here are some examples you can refer to. Including the below connectors, you can
     <td>hive_password</td>
   </tr>
   <tr>
-    <td>hive.proxy.user</td>
-    <td>true or false</td>
+    <td>default.proxy.user.property</td>
+    <td>hive.server2.proxy.user</td>
+  </tr>
 </table>
 
 Connection to Hive JDBC with a proxy user can be disabled with `hive.proxy.user` property (set to true by default)
```

Comment:
is `hive.proxy.user` removed then? do we need to doc to explain the migration?
also you have this line here still with `hive.proxy.user property (set to true by default)`